windows-lb/datasources.tf

```diff
@@ -19,40 +31,12 @@ data "oci_limits_services" "compute_services" {
     values = ["compute"]
   }
 }
-data "oci_limits_limit_definitions" "compute_limit_definitions" {
```

Comment:
Why remove this ? We are a choking if the compute shape is intact available in the AD requested.
Not all shapes will be available in all ADs especially for free trial users who have not upgraded

Reply:
@jeevanjoseph ok, that makes sense. I will revert this change.